indy-credx/src/services/helpers.rs

```diff
@@ -7,8 +7,8 @@ use indy_data_types::anoncreds::{
 };
 
 use crate::anoncreds_clsignatures::{
-    hash_credential_attribute, CredentialSchema, CredentialValues as CryptoCredentialValues,
-    Issuer as ClIssuer, MasterSecret as CryptoMasterSecret, NonCredentialSchema, SubProofRequest,
+    hash_credential_attribute, CredentialSchema, CredentialValues as ClCredentialValues,
+    Issuer as ClIssuer, MasterSecret as ClMasterSecret, NonCredentialSchema, SubProofRequest,
```

Comment:
Did you mean to leave this reference to MasterSecret here?

Reply:
It's still called MasterSecret in the anoncreds-clsignatures library, that can be fixed in another release.